docs/web/experimental.css

```diff
@@ -115,203 +95,52 @@ img#pfp {
         margin-bottom: 0;
 }
 
-a:hover {
-  color: #d997ff;
-  text-decoration: underline;
-  }
+ul#sidelist {
+    background-color: transparent;
+    font-size: 20px;
+    border: none;
+  line-height:2.0em;
+  font-family: 'Acme';
+  padding-right: 100px;
+}
 
-.welcome{
-    text-align:center;
-    margin-left: auto;
-    margin-right: auto;
+  div.main{
+    /*  flex:4;*/
+    padding:1vw;
+    /*margin-left:auto;
+    margin-right: auto;*/
+    display: grid;
+    
 }
 
-h2.welcome {
-    font-family: 'Sriracha', cursive;
+img{
+display: grid;
```

Comment:
@arrowarchive Typically an `<img>` file is nested deep inside container elements. It is the *container* elements that take the `display:grid` CSS property. You need to set up a grid on a container element higher up the tree.

Reply:
@arrowarchive This is to say, I believer your `display:grid` properties are sitting on the wrong elements in this CSS file.